rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -309,6 +310,37 @@
         }
     }
 
+    /**
+     * Returns a list of jobs info corresponding to the given job IDs (in the same order)
+     *
+     * @param jobsId
+     *            the list of id of the jobs to return, in the same order
+     * @param sessionId
+     *            a valid session id
+     * @return a list of UserJobData
+     */
+    @Override
+    @GET
+    @Path("jobsinfolist")
+    @Produces({ "application/json", "application/xml" })
+    public List<UserJobData> jobsInfoList(@HeaderParam("sessionid") String sessionId,
+            @QueryParam("jobsid") List<String> jobsId)
+            throws PermissionRestException, NotConnectedRestException, UnknownJobRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)